src/Map/index.js

```diff
@@ -1,2 +1,2 @@
-export { default } from './Map'
-export { connectToMap, createMap, createSelectInteraction, updateMapFromUrl, updateUrlFromMap } from './utils'
+export { default as Map } from './Map'
+export * from './utils'
```

Comment:
this goes along with all the `*` exports from `src/index.js`...all default exports need to be named in their directory's `index.js` file like above (instead of previously being named in `src/index.js`)